developerportal/apps/events/models.py

```diff
@@ -13,6 +13,7 @@
 )
 from django.utils.safestring import mark_safe
 
+from django_countries.fields import CountryField
```

Comment:
How come this is added during this PR? Was it absent before and only working because of a lucky import elsewhere?

Reply:
This import was grouped incorrectly by the author and isort picked it up (see line 33)

Reply:
🤦‍♂ Thanks!